tests/test_scheduler.py

```diff
@@ -223,3 +224,98 @@ def setup(self): # celery3
     def test_sentinel_scheduler(self):
         redis_client = get_redis(app=self.app)
         assert 'Sentinel' in str(redis_client.connection_pool)
+
+class SSLConnectionToRedis(AppCase):
+
+    config_dict = {
+            'REDBEAT_KEY_PREFIX': 'rb-tests:',
+            'REDBEAT_REDIS_URL': 'rediss://redishost:26379/0',
+            'REDBEAT_REDIS_OPTIONS': {
+                'password': '123',
+                'service_name': 'master',
+                'socket_timeout': 0.1,
+            },
+            'CELERY_RESULT_BACKEND' : 'redis-sentinel://redis-sentinel:26379/1',
+            'REDBEAT_REDIS_USE_SSL': {
```

Comment:
Why are the ```ssl_*``` options in a new config key rather than folding them into ```REDBEAT_REDIS_OPTIONS```?

Reply:
Yeah, I thought about that but then figured I wanted to mirror the options for the Broker.  From the looks at the README and broker options, `BROKER_URL <=> REDBEAT_REDIS_URL`, `BROKER_TRANSPORT_OPTIONS <=> REDBEAT_REDIS_OPTIONS` -- so I figured having `BROKER_USE_SSL <=> REDBEAT_REDIS_USE_SSL` was the natural way to go.  Happy to change it if you think adding them to `REDBEAT_REDIS_OPTIONS` is better.

Reply:
Thanks I see your logic now, and agree it follows the pattern established upstream

.  It might be worth evaluating if we actually want to support ```REDBEAT_REDIS_USE_SSL=True``` or not.  From the [celery docs](http://docs.celeryproject.org/en/latest/userguide/configuration.html#broker-use-ssl) that could allow users a false sense of security and forcing them to be explicit would avoid security mistakes.  What do you think?

Reply:
Hmm, well, I'm relatively new to python, so not sure what the better convention here would be.  In our case, I made sure that if `REDBEAT_REDIS_USE_SSL` was equal to `True`, that we were verifying the certificate by having `ssl.CERT_REQUIRED` be in the default dictionary.  So in terms of security safety, I think we'd be covered.

I can see arguments for either case regarding conventions though, on one hand, setting the value to `True` is an anti-pattern since not _every_ library guarantees that it'll verify certificates and we could actively discourage this pattern by not allowing it, but on the other hand, allowing `True` but in a safe way makes this library more consistent with other libraries that are like it.

Since it's more of a convention issue, I'm happy to yield to whatever you think is best here.

Reply:
As per my comment on https://github.com/sibson/redbeat/pull/110/files#diff-3306d7d435bbd01ef84d83cae122c8cfR121, it seems like we won't be able to protect users from themselves by forcing CERT_REQUIRED.  It seems the best we can hope to do is document the best practice.

Reply:
Hmmm, sure, I'll add a bit in the README similar to what the celery docs have for the broker.